nexus/tests/integration_tests/instances.rs

```diff
@@ -1927,77 +1927,6 @@ async fn test_attach_one_disk_to_instance(cptestctx: &ControlPlaneTestContext) {
     assert_eq!(disks[0].state, DiskState::Attached(instance.identity.id));
 }
 
-#[nexus_test]
-async fn test_instance_fails_to_boot_with_disk(
```

Comment:
FYI to @jmpesp , this was removed, but I believe we have coverage with the other idempotency tests.